components/calendar/src/cal/hijri.rs

```diff
@@ -58,9 +57,16 @@ fn year_as_hijri(standard_era: tinystr::TinyStr16, year: i32) -> types::YearInfo
 ///
 /// This calendar is a pure lunar calendar with no leap months. It uses month codes
 /// `"M01" - "M12"`.
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 pub struct HijriObservational {
-    data: Option<DataPayload<CalendarHijriObservationalV1>>,
+    pub(crate) location: HijriObservationalLocation,
+    data: Option<DataPayload<ErasedMarker<HijriCache<'static>>>>,
+}
+
+#[derive(Clone, Debug, Copy, PartialEq)]
+pub enum HijriObservationalLocation {
+    Cairo,
+    Mecca,
```

Comment:
Probably use the spelling Makkah. It is the official Saudi spelling for 40 years and it sounds more like the Arabic pronunciation.

https://www.arabnews.com/node/1537906/saudi-arabia

Reply:
Should Cairo be al-Qāhirah?

(we can always just use Arabic identifiers if we really want)